config/routes.rb

```diff
@@ -22,7 +22,7 @@ def set_up_flipper
 end
 
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, controllers: { sessions: "users/sessions" }
```

Comment:
Could you explain why this was added @js-sapphire ? I'am not sure why this was necessary?

Reply:
This was necessary so that SessionsController would get called for the before_action **check_failed_login** to run